daniel-alejandro-garcia/tarea-11/to-do-list-routing/src/App.js

```diff
@@ -0,0 +1,26 @@
+
+import { Link } from 'react-router-dom';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <h1>ToDo list Routing</h1>
+      <ul>
+        <li><Link to="/taskOne">Magazine Cover</Link></li>
```

Comment:
@da-garcia , this route list should be dynamic, and built from an array using map function